### ISSUE TITLE
Not working properly with dark theme

### ISSUE THREAD

**Issue by matiast1905** [xiaohk/stickyland#7]
Hi!
This is a great extension, really useful!
I want you to know that it's not displaying the proper text output with the jupyter lab dark theme.
It's showing the text in the same color of the background. I think the problem is that the background of the widget should be dark as well, so we could read the white text of the output.
Here goes a screenshot:

![image](https://user-images.githubusercontent.com/65732266/159103917-100b21af-563f-4089-a9ce-f82709acb5ec.png)
![image](https://user-images.githubusercontent.com/65732266/159103942-9cf3c419-fd9b-4d28-8c4d-c954b142b838.png)

As you can see, I had to select the text in order to read the output

Thank you!

**Comment by gingsi** [comment]
Hi, had the same problem. A quick fix is to modify jupyterlab's dark theme.

1. Find the style sheet, mine was in `$CONDA_INSTALL_DIR/envs/$ENV_NAME/share/jupyter/lab/themes/@jupyterlab/theme-dark-extension/index.css` (replace the two variables accordingly)
2. At the end of the file add ` .sticky-content .content {background-color:var(--jp-layout-color0) !important;}`
3. Reload browser, make sure to delete the cache to get the new css.

This is not perfect (e.g. buttons are still white on white and not readable), however it works well enough for me.

**Comment by matiast1905** [comment]
Hi @gingsi, 
This is a nice get around for the problem! As you mention, it's not perfect but it's really usable changing just that.
Thanks for the advice!

**Comment by xiaohk** [comment]
Hello @matiast1905, thank you so much for raising this issue! I didn't know JupyterLab supports dark theme out-of-the box. It is so cool!

Thank you @gingsi for your workaround. I just realized with CSS variables, JupyterLab will automatically handle light theme and dark theme colors. 🤩🤩

Based on your suggestion, I have updated StickyLand CSS files to support the dark theme in https://github.com/xiaohk/stickyland/commit/877777d2480b4825abd45c60e78ad81d8d220f4b.

This is included in the [new release v0.2.1](https://github.com/xiaohk/stickyland/releases/tag/v0.2.1).


https://user-images.githubusercontent.com/15007159/160261625-27ad33f0-a2a9-435a-bd90-1fc10846b8f1.mp4
